### PR TITLE
feat(docs): Connect UI Documentation 

### PR DESCRIPTION
**Pull request by dannylwe**
Add docs for:
- gong 
- greenhouse-basic
- hibob-service-user
- klaviyo
- lever-basic
- marketo
- personio
- sage-intacct 
- woocommerce
- workable
- workday
- zoominfo

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -2768,10 +2768,25 @@ gong:
     proxy:
         base_url: https://api.gong.io
     docs: https://docs.nango.dev/integrations/all/gong
+    docs_connect: https://docs.nango.dev/integrations/all/gong/connect
+    credentials:
+        username:
+            type: string
+            title: username
```
Comment (khaliqgant) [inline packages/shared/providers.yaml]:
```suggestion
            title: Access Key
```

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -2768,10 +2768,25 @@ gong:
...
+            description: Your Gong Access Key
+            pattern: '^[a-zA-Z0-9-]+$'
+        password:
+            type: string
+            title: Password
```
Comment (khaliqgant) [inline packages/shared/providers.yaml]:
```suggestion
            title: Access Key Secret
```

File: docs-v2/integrations/all/greenhouse-basic/connect.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - API type: Job board
+    - Description: Give your API key a descriptive name
+
+5. When finished, click `Create` to generate the API key.
+
+6. After the key is created, click `Copy`. Paste the API key in a secure location.
+
+7. Click `I have stored the API Key`.
+
+8. Note your greenhouse.io domain in your browser.
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/greenhouse-basic/connect.mdx:39]: The steps are not linked in `providers.yaml` and it's missing the step 1 about the domain

File: docs-v2/integrations/all/greenhouse.mdx [hunk 4]
```diff
@@ -34,3 +34,7 @@ Greenhouse offers both `Basic` and `OAuth` as authentication and Nango implement
 - If you need to use the proxy, it is important to provide the resource that you will be calling in the config (as green house has many forms for API resource). Please see [here](https://developers.greenhouse.io/) for different types of resources. Once confirmed, you have to add `resource` as a config.
 
 <Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/greenhouse.mdx)</Note>
+
+<Card title="Connect to grennhouse" icon="link" href="/integrations/all/greenhouse-basic/connect" horizontal>
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/greenhouse.mdx]:
```suggestion
<Card title="Connect to Greenhouse" icon="link" href="/integrations/all/greenhouse-basic/connect" horizontal>
```

File: docs-v2/integrations/all/hibob.mdx [hunk 5]
```diff
@@ -30,3 +30,7 @@ API configuration: [`hibob`](https://nango.dev/providers.yaml)
 - For Basic auth, HiBob uses ID as the username and token as the password.
 
 <Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/hibob.mdx)</Note>
+
+<Card title="Connect to hibob" icon="link" href="/integrations/all/hibob-service-user/connect" horizontal>
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/hibob.mdx]:
```suggestion
<Card title="Connect to Hibob" icon="link" href="/integrations/all/hibob-service-user/connect" horizontal>
```

File: docs-v2/integrations/all/klaviyo/connect.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+To authenticate with Klaviyo, you need one key piece of information:
+1. **API Key** - Your Klaviyo private key.
+
+This guide will walk you through finding or creating those credentials within Klaviyo.
+
+### Prerequisites:
+
+- You must set up an account with Klaviyo.
+
+#### Step 1: Finding Klaviyo API Key
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/klaviyo/connect.mdx]:
```suggestion
#### Step 1: Finding your Klaviyo API Key
```

File: docs-v2/integrations/all/lever.mdx [hunk 7]
```diff
@@ -31,3 +31,7 @@ API configuration: [`lever`](https://nango.dev/providers.yaml), [`lever-basic`](
 - You should create different integrations for using the Lever production vs. sandbox API.
 
 <Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/lever.mdx)</Note>
+
+<Card title="Connect to lever-basic" icon="link" href="/integrations/all/lever-basic/connect" horizontal>
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/lever.mdx]:
```suggestion
<Card title="Connect to Lever" icon="link" href="/integrations/all/lever-basic/connect" horizontal>
```

File: docs-v2/integrations/all/marketo/connect.mdx [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+1. **Client ID** - Your Marketo Access Client ID
+2. **Client Secret** - Your Marketo Client Secret
+
+This guide will walk you through finding or creating those credentials within Marketo.
+
+### Prerequisites:
+
+- You must set up an account with Marketo.
+
+#### Step 1: Finding Marketo API Key and API Key Secret
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/marketo/connect.mdx]: split in two

File: docs-v2/integrations/all/marketo/connect.mdx [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+#### Step 2: Connect Your Marketo Account
+
+<img src="/integrations/all/gong/nango_connect_ui.png" />
+
+To authenticate using your API credentials, navigate to the Marketo authentication form. Enter your credentials as follows:
+- Client ID: Your Marketo Access Key
+- Client Secret: Your Marketo Access Key Secret
+- Domain: Your Marketo domain (xxx.mktorest.com)
+- Identity URL: Your Marketo domain (xxx.mktorest.com/identity/oauth/token)
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/marketo/connect.mdx]: This should be in dedicated steps

**Review by bodinsamuel — COMMENTED** [review]
Huge 💪🏻 (Shipping 10 will make this review a bit complex, we talked with khaliq to improve this next time)
I haven't read everything, here some global feedback:
- Each field in the Connect UI should ideally be one section in the markdown 
- Missing doc_section on every provider in providers.yaml
- Brand casing
- Section title consistency: "Finding your X" works well I think, but maybe we should decide on a guideline here

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**